torchci/rockset/commons/__sql/flaky_workflows_jobs.sql

```diff
@@ -1,16 +1,33 @@
--- Find workflows that succeeded once they were retried
 with repeats as (
     select
-        j.run_id,
-        j.name
+        array_agg(j.id) as ids
     from
         workflow_job j
+        join workflow_run w on w.id = j.run_id
     where
         j._event_time >= PARSE_DATETIME_ISO8601(:startTime)
         and j._event_time < PARSE_DATETIME_ISO8601(:stopTime)
-        and j.run_attempt > 1
-        and j.conclusion = 'success'
-) -- When a first time contributor submits a PR, their workflow starts at "attempt 2" so we look for workflows that were repeated above and actually have a first attempt
+        and w.repository.full_name = :repo
+        and w.head_branch = :branch
+        AND w.event != 'workflow_run'
```

Comment:
A qq: Besides push and schedule events, are there any other events showing up on HUD that we want to query for here?

Reply:
I think those are the two that matter most for the main page of hud and master related stats, but I took this from hud_query and I'm not sure if hud_query is used for other stuff like querying for pr jobs where pull_request event would be needed